src/pages/data-management/index.jsx

```diff
@@ -173,11 +155,19 @@ const DataManagementPage = ({ route }) => {
         route={route}
         title='Data Management'
       />
-      <LoadingModal
-        visible={Boolean(projectSaving || sampleSaving)}
-        message={projectSaving ?? sampleSaving ?? ''}
-      />
-
+      {projectSaving || sampleSaving ? (
+        <center>
+          <Space direction='vertical'>
+            <ClipLoader
+              size={50}
+              color='#8f0b10'
+            />
+            <Text>
+              Loading...
+            </Text>
+          </Space>
+        </center>
+      ) : (<></>)}
```

Comment:
take a look whether we should reuse the Loader component

Reply:
We can't reuse the Loader component -- the Loader component is specifically designed for when we have experimentId, because it requires it as props.